File: serverTCC/serverTCC/Models/Configuracoes.cs
namespace serverTCC.Models
{
    public enum Idioma
    {
        Portugues,
        Ingles
    }

    public class Configuracoes
    {
        public int Id { get; set; }
        public string UsuarioId { get; set; }
        public Idioma Idioma { get; set; }
        public Usuario Usuario { get; set; }
    }
}

File: client/www/js/models/configuracoes.js
function Configuracoes(data) {
    "use strict";
    if (!data) data = {};

    this.id = data.id;
    if (data.usuario)
        this.usuario = new Usuario(data.usuario);
    this.usuarioId = data.usuarioId;
    this.idioma = data.idioma;
}

localEntities.register(Configuracoes, {
    id: {
        pk: true,
        type: 'number'
    },
    usuario: {
        relationKey: 'usuarioId',
        type: 'Usuario'
    },
    usuarioId: {
        fk: true,
        type: 'number'
    },
    idioma: {
        type: 'number'
    }
});